src/utils.js

```diff
@@ -392,3 +393,5 @@ export function combineURLs(baseURL, relativeURL) {
 export const isPushNotificationAvailableForUser = (state) => (
   state.pushNotification.available && state.pushNotification.deviceRegistered
 );
+
+export const getTimestampFormat = (timestamp) => moment.unix(timestamp).format(t`DD MMM YYYY [•] HH:mm`)
```

Comment:
Missing docstring

Reply:
Also, is this localized?

Reply:
It is internationalized but regarding localization for `en-us` I didn't prepared it, I'm following the pattern we already have. Maybe we should open an issue just for this.

Reply:
Can you please create the issue and link here?